apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -70,23 +78,43 @@ def __str__(self):
 
 
 class MeasTouschekLifetime(_BaseClass):
-    """."""
+    """Measurement of two single-bunches current decay with BPM sum signal.
+
+    In single-bunch mode, RFFE attenuation must be adjusted in all BPMs
+    before injection, the default value is 30dB. The BPM used to be measured
+    is allowed to have an attenuation of 14dB to increase the sum signal. In
```

Comment:
`increase the sum signal, if the maximum single-bunch current is bellow XXmA`

Reply:
ok, with 3.0mA and attenuation of 14dB it was observed that the sum signals are within the GCA recommendations. I added this info in the docstring as suggested.